force-app/main/default/classes/GetProcessInstanceDataTest.cls

```diff
@@ -1,95 +1,151 @@
 @isTest
 public with sharing class GetProcessInstanceDataTest {
 
-   
-   
-    public static void setup() {
-        //create a record
-        // Insert an account
-        Account curAccount = new Account(Name='Test',annualRevenue=100.0);
-        insert curAccount;
+    @TestSetup
+    private static void makeData(){
+        //insert a record that has an approval process on the targeted object
+        Account account = new Account();
```

Comment:
Insert the record of the type sObject that has an active approval we can submit to.